qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/java/QuteErrorCode.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+* Copyright (c) 2021 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.jdt.internal.java;
+
+public enum QuteErrorCode implements IQuteErrorCode {
+
+	NoMatchingTemplate("No template matching the path {0} could be found for: {1}");

```

Comment:
remove the `:`

Reply:
I did that, to follow the same message than Quarkus.

Reply:
ok then